apiServer.js

```diff
@@ -87,6 +96,57 @@ async function createDnsCache(userCache) {
   }
 }
 
+function setAxiosDefaults(socketIoServers) {
+  if (axiosDefaultsSet) return;
+
+  axiosDefaultsSet = true;
+
+  log.info('setting axios defaults');
+  axios.defaults.timeout = 20_000;
```

Comment:
is the numeric separator _ supported in nodejs 14?

Reply:
Yes - have tested.